src/Exceptions/InvalidCurrencyException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions;
+
+class InvalidCurrencyException extends \LogicException
+{
+    public function __construct($currency)
+    {
+        sprintf('Unknown format value: "%s"', $currency);
```

Comment:
What does this line do? Aren't you missing a call to the parent constructor or something?